networkx/algorithms/community/louvain.py

```diff
@@ -0,0 +1,325 @@
+"""Function for detecting communities based on Louvain Community Detection
+Algorithm"""
+
+from collections import deque, defaultdict
+
+import networkx as nx
+from networkx.algorithms.community import modularity
+from networkx.utils import py_random_state
+
+__all__ = ["louvain_communities", "louvain_partitions"]
+
+
+@py_random_state("seed")
+def louvain_communities(
+    G, weight="weight", resolution=1, threshold=0.0000001, seed=None
+):
+    r"""Find the best partition of G using the Louvain Community Detection
+    Algorithm.
```

Comment:
Just some documentation nit picks here:
Could you change this to `partition of a graph` instead of `partition of G`.